joomla/joomla-cms
# [5.1] Convert mod_custom to service provider

### Summary of Changes

Converts the custom administrator module to service provider.


### Testing Instructions

1. Make sure you have administrator module mod_custom published and have some content in WYSIWYG
2. You should see module output

![mod_custom](https://github.com/joomla/joomla-cms/assets/59166858/76e2271f-4ecc-48be-81a5-03e81f809331)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
